mlir/lib/ExecutionEngine/ExecutionEngine.cpp

```diff
@@ -451,3 +447,14 @@ Error ExecutionEngine::invokePacked(StringRef name,
 
   return Error::success();
 }
+
+void ExecutionEngine::initialize() {
+  if (isInitialized) {
+    return;
+  }
```

Comment:
```suggestion
  if (isInitialized)
    return;
```

Nit: no trivial braces.